packages/flutter_hooks/test/use_callback_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+import 'mock.dart';
+
+void main() {
+  testWidgets(
```

Comment:
I like that we make a new file for this. But could you move existing useCallback tests in here too then?
https://github.com/rrousselGit/flutter_hooks/blob/2a6b45cc5c893e14da69cab7485047c7f1e30f2f/packages/flutter_hooks/test/memoized_test.dart#L68